flow/designs/asap7/mock-array/rules-base.json

```diff
@@ -48,15 +48,15 @@
         "compare": "<="
     },
     "finish__timing__setup__ws": {
-        "value": 0.0,
+        "value": -15.92,
```

Comment:
Any insight into this degradation?

Reply:
I think we need the --annealing io fix first before we have something comparable, but that said things are quite different w.r.t. routing and metal layers. 15ns isnt much, CTS has more variations, etc

Reply:
I hope its ps